### PR TITLE
submitted an initial solution to week 9

### PR DESCRIPTION
**Pull request by Andrey8860**

### CODE REVIEW

File: Week 9/solution/src/test/java/com/epam/week9/driver/Driver.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	private static WebDriver driver;
+	protected final Logger logger = LogManager.getRootLogger();
+
+	private Driver() {
+
+	}
+
+	public static WebDriver getDriver() {
+		if (driver == null) {
+			switch (System.getProperty("browser")) {
```
Comment (mamax) [inline Week 9/solution/src/test/java/com/epam/week9/driver/Driver.java:22]: Where property `browser` is set up?

File: Week 9/solution/src/test/java/com/epam/week9/service/EmailCreator.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.epam.week9.service;
+
+import com.epam.week9.model.Email;
+
+public class EmailCreator {
+	
+	public static Email emptyEmail() {
```
Comment (mamax) [inline Week 9/solution/src/test/java/com/epam/week9/service/EmailCreator.java:7]: this method is not used in project

File: Week 9/solution/src/test/java/com/epam/week9/driver/Driver.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+				}
+				default: { 
+					WebDriverManager.chromedriver().setup();
+					driver = new ChromeDriver();
+					break;
+				}
+			}
+			driver.manage().window().maximize();
+		}
+		return driver;
```
Comment (mamax) [inline Week 9/solution/src/test/java/com/epam/week9/driver/Driver.java:41]: The project is failing with NullPointerException

File: Week 9/solution/src/test/java/com/epam/week9/driver/Driver.java (hunk 1 above)
Comment (mamax) [reply Week 9/solution/src/test/java/com/epam/week9/driver/Driver.java:22]: you can create some `config.properties` file and set up what browser, env, etc by default

Comment (Andrey8860) [reply Week 9/solution/src/test/java/com/epam/week9/driver/Driver.java]: Resolved

File: Week 9/solution/src/test/java/com/epam/week9/service/EmailCreator.java (hunk 2 above)
Comment (Andrey8860) [reply Week 9/solution/src/test/java/com/epam/week9/service/EmailCreator.java]: Resolved

**Pull request merged**